src/pt/mangaonline/build.gradle

```diff
@@ -0,0 +1,13 @@
+ext {
+    extName = 'Manga Online'
+    extClass = '.MangaOnline'
+    extVersionCode = 1
+    isNsfw = true
+}
+
+apply from: "$rootDir/common.gradle"
+
+dependencies {
+    implementation(project(':lib:randomua'))
+}
+
```

Comment:
remove extra newline, only one is needed

Reply:
Resoved: https://github.com/keiyoushi/extensions-source/pull/1566/commits/893e5cc5c678d7c75197738dd0157c805ac41a2d. Following madara build.kts as example